keyboards/keychron/k3/keymaps/iso/keymap.c

```diff
@@ -21,13 +21,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // Layer names don't all need to be of the same length, obviously, and you can also skip them
 // entirely and just use numbers.
 enum layer_names {
-  _BASE = 0,
-  _FN,
+  MAC_BASE = 0,
+  WIN_BASE,
+  MAC_FM,
+  WIN_FM
 };
 
+#define KC_TASK LGUI(KC_TAB)        // Task viewer
+#define KC_FLXP LGUI(KC_E)          // Windows file explorer
+#define KC_SNIP LGUI(LSFT(KC_S))    // Windows snip tool
+
+#define KC_MSSN LGUI(KC_F3)         // Mission Control
+#define KC_FIND LALT(LGUI(KC_SPC))  // Finder
+#define KC_MSCR LSFT(LGUI(KC_3))    // Mac screenshot
+#define KC_MSNP LSFT(LGUI(KC_4))    // Mac snip tool
+
```

Comment:
Copying keychron Q1 here? Nice

Reply:
Yes, I was using process_record_user, host_consumer_send(...) for mission control and launchpad but then i saw that Q1 already handle those keys and I just copy/paste the same defs.